content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md

```diff
@@ -182,7 +182,7 @@ For the new node `backend-4` with IP address `10.0.0.4`:
 etcd-advertise-client-urls: "http://10.0.0.4:2379"
 etcd-listen-client-urls: "http://10.0.0.4:2379"
 etcd-listen-peer-urls: "http://0.0.0.0:2380"
-etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380,backend-4=https://10.0.0.4:2380"
+etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380,backend-4=http://10.0.0.4:2380"
```

Comment:
I noticed that some of our code examples seem to mix http and https...I think http is correct at this point (later instructions discuss securing cluster members) but please correct me if not.